inc/Engine/Admin/Settings/Settings.php

```diff
@@ -693,7 +693,7 @@ private function check_global_exclusion( $field ) {
 	private function sanitize_cdn_cnames( array $cnames ) {
 		$cnames = array_map(
 			function( $cname ) {
-				return filter_var( $cname, FILTER_VALIDATE_DOMAIN, FILTER_FLAG_HOSTNAME );
+				return filter_var( $cname, FILTER_VALIDATE_DOMAIN );
```

Comment:
not using the hostname flag doesn't filter anything, it will validate incorrect outputs.

Other solution is to use the filter only on the host part, by slicing the url with parse_url and reconstructing it after

Reply:
Done